tests/stub/shared.py

```diff
@@ -28,6 +29,8 @@
     INTERRUPT_EXIT_CODE = -signal.SIGINT
     POPEN_EXTRA_KWARGS = {}
 
+clean_ports = set()
```

Comment:
Would not detect trying to bind two stub servers to the same port. Not sure if needed :shrug: